Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md

```diff
@@ -0,0 +1,275 @@
+# Microsoft Sentinel Solutions Packaging Tool Guidance For CCP Connector
```

Comment:
I am unsure about the folder placing of ccp contents. check this with Prateek

Reply:
as per discussion with Prateek added a link in v3 file and kept in the same location where v3 file is.